server/sonar-search/src/main/java/org/sonar/search/EsSettings.java

```diff
@@ -149,6 +149,31 @@ private InetAddress readHost() {
     }
   }
 
+  private String readInitialStateTimeout() {
+    String initialStateTimeout = props.value(ProcessProperties.SEARCH_INITIAL_STATE_TIMEOUT, "30s").toLowerCase().trim();
+
+    if (initialStateTimeout.endsWith("ms") ||
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1067)